### ISSUE TITLE
REQUEST add GameShark codes and Code Breaker codes

### ISSUE THREAD

**Issue by benderscruffy** [fleroviux/NanoboyAdvance#111]

**Comment by fleroviux** [comment]
Definitely wanna add support for cheat codes. I think it makes sense to implement #113 first? So we can have a simple GUI for managing cheats from the get go.

**Comment by legendarysnake0** [comment]
this is the best gba emulator ever made, the sound is not strident or clunky like in another emulators. everything about it is perfect, except... well it needs a cheat support badly. im playing mother 1, and sometimes i just want to go explore the (huge)map without several random encounters. A no random cheats would be perfect for it. 
please add a cheat support mister Assignees  "Patchouli Knowledge" (i like touhou too)

**Comment by fleroviux** [comment]
@legendarysnake0 I agree that cheat support is one of the bigger/more important missing features at the moment and it definitely is at the back of my head. That being said, currently my focus is still making the core more accurate (1.7 will be a big update in that regard). Cheats is a topic I have worked a little bit on last year, but still am far away from anything shippable. Maybe it is something I will work on after 1.7 is released.
